resolwe_bio/docker_images/rnaseq/assets/genialis.css

```diff
@@ -0,0 +1,135 @@
+color: green;
+  src: url('fonts/roboto-v16-latin-ext_latin-300.eot'); /* IE9 Compat Modes */
+  src: local('Roboto Light'), local('Roboto-Light'),
+       url('fonts/roboto-v16-latin-ext_latin-300.eot?#iefix') format('embedded-opentype'), /* IE6-IE8 */
+       url('fonts/roboto-v16-latin-ext_latin-300.woff2') format('woff2'), /* Super Modern Browsers */
+       url('fonts/roboto-v16-latin-ext_latin-300.woff') format('woff'), /* Modern Browsers */
+       url('fonts/roboto-v16-latin-ext_latin-300.ttf') format('truetype'), /* Safari, Android, iOS */
+       url('fonts/roboto-v16-latin-ext_latin-300.svg#Roboto') format('svg'); /* Legacy iOS */
+}
+
+/* roboto-regular - latin-ext_latin */
+@font-face {
+  font-family: 'Roboto';
+  font-style: normal;
+  font-weight: 400;
+  src: url('fonts/roboto-v16-latin-ext_latin-regular.eot'); /* IE9 Compat Modes */
+  src: local('Roboto'), local('Roboto-Regular'),
+       url('fonts/roboto-v16-latin-ext_latin-regular.eot?#iefix') format('embedded-opentype'), /* IE6-IE8 */
+       url('fonts/roboto-v16-latin-ext_latin-regular.woff2') format('woff2'), /* Super Modern Browsers */
+       url('fonts/roboto-v16-latin-ext_latin-regular.woff') format('woff'), /* Modern Browsers */
+       url('fonts/roboto-v16-latin-ext_latin-regular.ttf') format('truetype'), /* Safari, Android, iOS */
+       url('fonts/roboto-v16-latin-ext_latin-regular.svg#Roboto') format('svg'); /* Legacy iOS */
+}
+
+/* roboto-italic - latin-ext_latin */
+@font-face {
+  font-family: 'Roboto';
+  font-style: italic;
+  font-weight: 400;
+  src: url('fonts/roboto-v16-latin-ext_latin-italic.eot'); /* IE9 Compat Modes */
+  src: local('Roboto Italic'), local('Roboto-Italic'),
+       url('fonts/roboto-v16-latin-ext_latin-italic.eot?#iefix') format('embedded-opentype'), /* IE6-IE8 */
+       url('fonts/roboto-v16-latin-ext_latin-italic.woff2') format('woff2'), /* Super Modern Browsers */
+       url('fonts/roboto-v16-latin-ext_latin-italic.woff') format('woff'), /* Modern Browsers */
+       url('fonts/roboto-v16-latin-ext_latin-italic.ttf') format('truetype'), /* Safari, Android, iOS */
+       url('fonts/roboto-v16-latin-ext_latin-italic.svg#Roboto') format('svg'); /* Legacy iOS */
+}
+
+/* roboto-500 - latin-ext_latin */
+@font-face {
+  font-family: 'Roboto';
+  font-style: normal;
+  font-weight: 500;
+  src: url('fonts/roboto-v16-latin-ext_latin-500.eot'); /* IE9 Compat Modes */
+  src: local('Roboto Medium'), local('Roboto-Medium'),
+       url('fonts/roboto-v16-latin-ext_latin-500.eot?#iefix') format('embedded-opentype'), /* IE6-IE8 */
+       url('fonts/roboto-v16-latin-ext_latin-500.woff2') format('woff2'), /* Super Modern Browsers */
+       url('fonts/roboto-v16-latin-ext_latin-500.woff') format('woff'), /* Modern Browsers */
+       url('fonts/roboto-v16-latin-ext_latin-500.ttf') format('truetype'), /* Safari, Android, iOS */
+       url('fonts/roboto-v16-latin-ext_latin-500.svg#Roboto') format('svg'); /* Legacy iOS */
```

Comment:
My browser console has an issue with parsing some `*-500` files.
Can you check your console / re-download them from google / re-commit

Reply:
The issue with *.500 files is fixed, thanks.